cmd/btfs/daemon.go

```diff
@@ -795,8 +797,18 @@ func functest(statusServerDomain, peerId, hValue string) {
 		test_success := false
 		// try up to two times
 		for i := 0; i < 2; i++ {
-			if err := get_functest(btfsBinaryPath); err != nil {
-				log.Errorf("BTFS daemon get file test failed!\n")
+			timer := time.NewTimer(timeoutSeconds * time.Second)
+			// if it's timeout, it will only do once
+			go func() {
+				<-timer.C
+				fmt.Println("BTFS daemon get file test timed out!")
+				os.Exit(getFileTestFailed)
```

Comment:
@tycm4109 

Do we expect the BTFS node exist when having timeout?
And for this scenario, the status server get no info?

Reply:
as explained in ticket https://tronnetwork.atlassian.net/browse/BTFSINFRA-500
exiting is the only feasible way to stop a hanging function. 
sending this info to status server worth considering - ticket created . 
https://tronnetwork.atlassian.net/browse/BTFSINFRA-514

Reply:
There must be a way, which should be trivial. Please follow up on this.